doc/040_backup.rst

```diff
@@ -453,12 +459,11 @@ environment variables. The following lists these environment variables:
 
     RCLONE_BWLIMIT                      rclone bandwidth limit
 
-In addition to restic-specific environment variables, the following system-wide environment variables
-are taken into account for various operations:
+See :ref:`caching` for the rules concerning cache locations.
```

Comment:
This should mention, that this is about the default cache location. If `RESTIC_CACHE_DIR` is set, then that has precedence.

Reply:
Changed to:

    See :ref:`caching` for the rules concerning cache locations when
    ``RESTIC_CACHE_DIR`` is not set.